cluster-autoscaler/simulator/scheduling/hinting_simulator.go

```diff
@@ -83,6 +84,7 @@ func (s *HintingSimulator) TrySchedulePods(clusterSnapshot clustersnapshot.Clust
 			break
 		}
 	}
+	klogx.V(4).Over(loggingQuota).Infof("There were also %v other logs that were capped.", -loggingQuota.Left())
```

Comment:
nit: Maybe clarify what kind of logs, or include some other more precise info (e.g. prefix with the name of the function). Log lines from other goroutines could get inserted in between and then it'd be hard to know what this line refers to.